src/app/forms/templates/test-records/section-templates/test/test-section-group1.template.ts

```diff
@@ -47,13 +47,8 @@ export const TestSectionGroup1: FormNode = {
             {
               name: 'testResult',
               label: 'Result',
-              editType: FormNodeEditTypes.RADIO,
-              options: [
-                { value: 'fail', label: 'Fail' },
-                { value: 'pass', label: 'Pass' },
-                { value: 'prs', label: 'PRS' },
-                { value: 'abandoned', label: 'Abandoned' }
-              ],
```

Comment:
By hiding the editType how does the user select to abandon the test?

Reply:
We don't need to cater for that scenario, abandoned tests will stay abandoned and there is no requirement to set tests as abandoned VTM